ci/docker-deploy.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+#
+# quick docker deploy
+#
+# optional dockerhub login
+export DOCKERHUB_LOGIN="${DOCKERHUB_LOGIN:-}"
+export DOCKERHUB_TOKEN="${DOCKERHUB_TOKEN:-}"
+
+export APP_NAME="${APP_NAME:-ghost}"
+export APP_BRANCH="${APP_BRANCH:-master}"
+export APP_URL="https://github.com/datalab-mi/${APP_NAME}/archive/refs/heads/${APP_BRANCH}.tar.gz"
+
+export USE_CURL=true
+
+# if authenticated repo
+if [ -n "${GITHUB_TOKEN}" ] ; then
+  curl_args=" -H \"Authorization: token ${GITHUB_TOKEN}\" "
+fi
+
+# if APP_ROLE defined use make up-${APP_ROLE}
+if [ -n "$APP_ROLE" ] ;then
```

Comment:
I understand that app_role can be use to start either ghost or sql docker containers separately. I'm wondering when you will need to use it in the CI, because I think that you need to deploy both containers in order to have the application running

Reply:
For the future (in case of different docker host)

Yes start can be done in different order.
ghost, wait until db is comming up (if i understand the log)

Reply:
Yes, that's the case
We could also add a dependency between containers in docker-compose file

Reply:
* dependencies in docker-compose work for all containers running in one docker host
* In the of case of running each containers on separate docker hosts, dependencies doesnt work and , start will fail.